packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -63,9 +80,19 @@ export function SwitchNetworksButton(
       >
         <SwitchNetworkButtonBorderTop />
         {isSmartContractWallet ? (
-          <ArrowDownIcon className="h-6 w-6 stroke-1 text-white" />
+          <ArrowDownIcon
```

Comment:
I think if we can only transfer to Arbitrum chains we should show the arrow down icon. Then we don't need opacity changes.